### PR TITLE
Add datashader

### PR DESCRIPTION
**Pull request by jbcrail**
Fixes https://github.com/conda-forge/staged-recipes/issues/4028

### CODE REVIEW

File: recipes/datashader/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/bokeh/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
```
Comment (jakirkham) [inline recipes/datashader/meta.yaml]: Any thoughts on adding `noarch: python`?

**Comment by jakirkham** [comment]
One question. Otherwise LGTM.

File: recipes/datashader/meta.yaml (hunk 1 above)
Comment (jbednar) [reply recipes/datashader/meta.yaml]: Happy to have noarch: Python; I strongly believe in noarch!

File: recipes/datashader/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+  description: |
+    Datashader is a data rasterization pipeline for automating the process of
+    creating meaningful representations of large amounts of data.
+  doc_url: https://datashader.readthedocs.io/
+  dev_url: https://github.com/bokeh/datashader
+
+extra:
+  recipe-maintainers:
+    - jbednar
```
Comment (jakirkham) [inline recipes/datashader/meta.yaml]: Also, just to confirm, happy to be a maintainer here, @jbednar?

File: recipes/datashader/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipes/datashader/meta.yaml]: Cool, well let's add that and then I think this is good to go. :)

File: recipes/datashader/meta.yaml (hunk 2 above)
Comment (jbednar) [reply recipes/datashader/meta.yaml]: Yes.

Comment (jakirkham) [reply recipes/datashader/meta.yaml]: Thanks for confirming.

**Comment by jakirkham** [comment]
Now that this is noarch, we only need to verify the CircleCI build is ok, which it is. Also this has already been reviewed and approved. So have gone ahead and merged it.

**Comment by jakirkham** [comment]
Also both of you should receive invites to join conda-forge. Accepting these will confer permissions for the relevant feedstocks that you are listed on as maintainers.

**Comment by jbednar** [comment]
Thanks for spearheading this!  It has been on my list but one of the many things I hadn't gotten around to.

**Pull request merged**